txtdirect_test.go

```diff
@@ -454,3 +454,39 @@ func TestRedirectE2e(t *testing.T) {
 		}
 	}
 }
+
+func TestConfigE2e(t *testing.T) {
+	tests := []struct {
+		url    string
+		txt    string
+		enable []string
+	}{
+		{
+			"https://e2e.txtdirect",
+			txts["_redirect.path.txtdirect."],
+			[]string{},
+		},
+		{
+			"https://path.txtdirect/test",
+			txts["_redirect.path.e2e.txtdirect."],
```

Comment:
I think we need to rework both the urls to be used and the keys for the DNS records. They don't make their intent as clear from first view.

Reply:
Yeah, I just wrote some random stuff for records and URLs to just test the functionality. we should discuss and chose nice URLs and right txt records.